scripts/beans.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```

Comment:
@XgamerTV Could you add some descriptions of the added experiments in the README.md?